curriculum/challenges/english/blocks/lecture-working-with-css-variables/672cf3ca326da9f63683e236.md

```diff
@@ -166,7 +166,7 @@ The `@property` rule also allows for more complex fallback scenarios. For instan
 
 In this case, if someone tries to set `--padding` to an invalid value, like a color, the browser will fall back to the initial value of `10px`, maintaining type safety.
 
-The browser support for the CSS @property rule has significantly improved and is considered widely available across all major modern browser engines (like Chrome, Edge, Opera, Firefox, and Safari). It achieved robust, cross-platform stability—supported by all browsers since early 2024—primarily through its inclusion in the "Baseline 2024 Newly Available" features, making it a reliable tool for production use today.
+The CSS @property rule is considered widely available across all major modern browser engines (like Chrome, Edge, Opera, Firefox, and Safari) since early-2024. If you want to support earlier browsers you need to provide fallback CSS as explained above.
```

Comment:
```suggestion
The CSS `@property` rule is considered widely available across all major modern browser engines (like Chrome, Edge, Opera, Firefox, and Safari) since early-2024. If you want to support earlier browsers you need to provide fallback CSS as explained above.
```